metrics/toxicity/requirements.txt

```
@@ -0,0 +1,4 @@
+git+https://github.com/huggingface/evaluate@{COMMIT_PLACEHOLDER}
+evaluate
+transformers
+datasets
```

I don't think we need evaluate twice and also datasets is already a dependency of evaluate.

```suggestion
transformers
```